docker/Dockerfile

```diff
@@ -1,18 +1,15 @@
 # To build this container, go to ESMValTool root folder and execute:
 # docker build -t esmvaltool:latest . -f docker/Dockerfile
-FROM continuumio/miniconda3
+FROM condaforge/mambaforge
 
-COPY ./environment.yml /src/ESMValTool/environment.yml
 WORKDIR /src/ESMValTool
-RUN conda update -y conda && conda install -y -c conda-forge mamba && mamba env create --name esmvaltool --file environment.yml && conda clean --all -y
+COPY environment.yml .
+RUN mamba update -y conda mamba pip && mamba env create --name esmvaltool --file environment.yml && conda clean --all -y
 
 # Make RUN commands use the new environment:
 SHELL ["conda", "run", "--name", "esmvaltool", "/bin/bash", "-c"]
 
-COPY ./esmvaltool/install/R /src/ESMValTool/esmvaltool/install/R
-RUN Rscript ./esmvaltool/install/R/setup.R
-
-COPY . /src/ESMValTool
-RUN pip install -e . && pip cache purge && esmvaltool install Julia
+COPY . .
```

Comment:
looks very funky - copy here to here?

Reply:
Copy from the build context (I think this corresponds to the current working directory when calling the `docker build` command, so that would be the root of this repository) to the current working directory (set to /src/ESMValTool a few lines above) in the container image. The documentation for COPY is available [here](https://docs.docker.com/engine/reference/builder/#copy).